src/gui/qgsmapcanvas.h

```diff
@@ -133,8 +133,8 @@ class GUI_EXPORT QgsMapCanvas : public QGraphicsView
     const QgsMapSettings &mapSettings() const SIP_KEEPREFERENCE;
 
     /**
-     * Sets the temporal controller, tQgsMapCanvasInteractionBlockerhis controller will be used to
-     * update the canvas temporal range.
+     * Sets the temporal controller, QgsMapCanvasInteractionBlocker
```

Comment:
```suggestion
     * Sets the temporal \a controller for this canvas.
     *
     * The controller will be used to update the canvas' temporal range.
     * \since QGIS 3.14
```